src/benchmarks/gc/GC.Infrastructure/Notebooks/Examples/GCAnalysisExamples.ipynb

```diff
@@ -19,19 +19,9 @@
      "languageId": "polyglot-notebook"
     }
    },
-   "outputs": [
-    {
-     "data": {
-      "text/html": [
-       "<div><div></div><div></div><div><strong>Installed Packages</strong><ul><li><span>Microsoft.Data.Analysis, 0.20.1</span></li><li><span>Microsoft.Diagnostics.Tracing.TraceEvent, 3.1.6</span></li><li><span>Newtonsoft.Json, 13.0.3</span></li><li><span>XPlot.Plotly, 4.0.6</span></li><li><span>XPlot.Plotly.Interactive, 4.0.7</span></li></ul></div></div>"
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
-    }
-   ],
+   "outputs": [],
    "source": [
-    "#r \"nuget: Microsoft.Diagnostics.Tracing.TraceEvent, 3.1.7\"\n",
+    "#r \"nuget: Microsoft.Diagnostics.Tracing.TraceEvent, 3.1.9\"\n",
```

Comment:
Why weren't the other notebooks updated with the latest version of TraceEvent?

Reply:
Ooops, they should be, I just missed.